### PR TITLE
feat(#484): Changed UI for tables accross all views

### LINKED ISSUE #484: Change tables style
### User story
Change tables style to match mockups.

### PR DESCRIPTION
**Pull request by KeeganWoods1**
##### _What existing problem does the pull request solve?_

Tables more closely follow the stakeholder's requirements for UI.

##### _Demonstrate the code is solid. </br> Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI._

BookingsSearch
MyBookings
Users
Roles
Rooms
blackout

##### _List all issues this PR resolves_
- closes #484, #335

##### _Complete these items before requesting reviews_
- [x] Static Analysis Rules Validated
- [x] All commits follow naming conventions

### Sub-tasks Checklist
- [x] UI Implementation
- [x] UI Test
- [x] Backend Implementation
- [x] Backend Tests
- [x] Integration Tests
- [x] System Tests
- [x] Documentation

### CODE REVIEW

**Review by Alex-Hayeur — COMMENTED** [review]
There are some tests to fix, but specifically

`Failed asserting that table [blackouts] matches expected entries count of 9. Entries found: 0.`

For the weekly booking test has been fixed, if you rebase from master it should work correctly.

File: resources/css/global.css [hunk 1]
```diff
@@ -13,6 +13,11 @@ h2 {
   margin: 0 auto 0 auto;
 }
 
+/*All purpose yellow background*/
+.bg-yellow {
+  background-color: #FFC82B;
+}
+
```
Comment (WindOfRussia) [inline resources/css/global.css:20]: you can probably get away without using bg-yellow and use tailwind's bg-yellow-<shade> helpers. ex: `bg-yellow-500`

File: resources/js/Components/Tables/RoleTable.vue [hunk 2]
```diff
@@ -4,40 +4,39 @@
...
+          <td class="text-center">{{role.name}}</td>
+          <td class="text-center">
             <jet-dropdown width="48">
               <template #trigger>
                 <button
-                  class="flex items-center text-sm font-medium text-gray-500 hover:text-gray-700 hover:border-gray-300 focus:outline-none focus:text-gray-700 focus:border-gray-300 transition duration-150 ease-in-out mx-auto"
+                  class="flex text-sm font-medium text-gray-500 hover:text-gray-700 hover:border-gray-300 focus:outline-none focus:text-gray-700 focus:border-gray-300 transition duration-150 ease-in-out"
                 >
-                  <div class="text-3xl">. . .</div>
+                  <div class="">Permissions</div>
```
Comment (WindOfRussia) [inline resources/js/Components/Tables/RoleTable.vue:21]: empty class tag is not needed here

File: resources/js/Components/Tables/RoomTable.vue [hunk 3]
```diff
@@ -1,51 +1,60 @@
...
-            class="btn submit-btn"
-            @click="toggleAdvancedFilters()"
-        >Additional Search</button>
+    <div class="table-filter-container mb-12 flex flex-row">
+      <div class="ml-3 mr-6">
+        <h3 class="font-black">ROOMS</h3>
+      </div>
+      <div class="mx-2 border shadow-md bg-yellow">
+        <input type="text" v-model="filter">
+          <i class="fas fa-search ml-1 mr-2"></i>
```
Comment (WindOfRussia) [inline resources/js/Components/Tables/RoomTable.vue]: replace with `em`s or try using hero-icons svg instead https://heroicons.com/: 

```
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
  <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z" />
</svg>
```

File: resources/js/Components/Tables/RoomTable.vue [hunk 4]
```diff
@@ -1,51 +1,60 @@
...
+      </div>
+      <!--vertical line-->
+      <div class="border mx-16">
+      </div>
+      <div class="mx-6">
+        <h3 class="font-black">ADVANCED</h3>
+      </div>
+      <div class="mx-2 border shadow-md bg-yellow min-w-24">
+        <button @click="toggleAdvancedFilters()">
+          <i class="fas fa-filter mx-2 max-w"></i>
```
Comment (WindOfRussia) [inline resources/js/Components/Tables/RoomTable.vue]: same as before

File: resources/js/Components/Tables/RoomTable.vue [hunk 5]
```diff
@@ -1,51 +1,60 @@
...
-            class="text-center lt-grey underline">
+            class="text-center underline">
             {{room.status}}
             </td>
-            <td class="text-center lt-grey">Reference</td>
-            <td class="lt-grey p-3">
-             <div class="text-md mx-2">
-               <jet-dropdown width="48">
+            <td class="">
+             <div class="">
```
Comment (WindOfRussia) [inline resources/js/Components/Tables/RoomTable.vue:54]: empty class tags should be removed

**Review by WindOfRussia — CHANGES_REQUESTED** [review]
I like this new search filter, definitely more polished than before. Requested a few changes before approving.

File: resources/js/Components/Tables/RoleTable.vue [hunk 6]
```diff
@@ -4,40 +4,39 @@
...
+          <td>
             <div class="text-md mx-2">
               <jet-dropdown width="48">
                 <template #trigger>
                   <button
-                  class="flex items-center text-sm font-medium text-gray-500 hover:text-gray-700 hover:border-gray-300 focus:outline-none focus:text-gray-700 focus:border-gray-300 transition duration-150 ease-in-out mx-auto"
+                  class="flex text-sm font-medium text-gray-500 hover:text-gray-700 hover:border-gray-300 focus:outline-none focus:text-gray-700 focus:border-gray-300 transition duration-150 ease-in-out"
                   >
-                    <div class="text-3xl">. . .</div>
+                    <div class="">Action</div>
```
Comment (WindOfRussia) [inline resources/js/Components/Tables/RoleTable.vue:39]: looks like you still got an empty class tag here

**Review by Alex-Hayeur — DISMISSED** [review]
The new tables are looking really good, plus nice catch on the js tests to remove the unnecessary assertion

**Review by WindOfRussia — CHANGES_REQUESTED** [review]
Overall good, still got the 2 empty class tags to remove.

**Review by WindOfRussia — APPROVED** [review]
Perfect! Let's get this merged!

**Review by Alex-Hayeur — APPROVED** [review]

**Pull request merged**